services/app-web/src/components/Form/FieldYesNo/FieldYesNo.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { useField } from 'formik'
+import { Fieldset } from '@trussworks/react-uswds'
+import { FieldRadio } from '../FieldRadio/FieldRadio'
+import { PoliteErrorMessage } from '../../'
+
+import styles from './FieldYesNo.module.scss'
+
+/**
+ * This component renders a pair of radio buttons with Yes and No as the options.
+ *
+ * It relies on the Formik useField hook to work, so it must ALWAYS be rendered
+ * inside of a Formik form context.
+ *
+ * If you want to use these components outside a Formik form, you can use the
+ * ReactUSWDS components directly.
+ */
+
+export type FieldYesNoProps = {
+    name: string
+    label: string
+    showError: boolean
```

Comment:
suggestion - make `showError?: boolean`and default to false rather than making this a required prop. i try to keep required props to things that would break the markdown or accessibility for these kinds of re-useable elements